python/overrides.js

```diff
@@ -99,4 +99,8 @@
   	'check region by element within shadow dom with vg': {skipEmit: true},
     'pageCoverage data is correct': {skipEmit: true},
     'pageCoverage data is correct with vg': {skipEmit: true},
+    'should abort after close': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8977981)
```suggestion
    "should abort after close": {skipEmit: true},
```